lib/cli/commands/unlock.ts

```diff
@@ -4,11 +4,11 @@ import { UnlockNodeRequest } from '../../proto/xudrpc_pb';
 
 export const command = 'unlock <password>';
 
-export const describe = 'unlock an xud node';
+export const describe = 'unlock your xud node';
```

Comment:
This seems a bit off to me because we're not using second person (you/your) elsewhere and it's not really what I'm used to with other command line tools` help, for example `lncli unlock` says "unlock an encrypted wallet." This could just say "unlock xud" perhaps.

Reply:
Alright, maybe a bit too much "product", fixed.